README.md

```diff
@@ -1,15 +1,13 @@
 # Straitjacket
 
-An opinionated, one file framework for writing more reasonable code in a
-functional style which quarantines complexity.
+A one-file framework for reasonable code in a functional style.
 
 
 ## Introduction
 
-Software, like any creative endeavor, is controversial. "Good" is eternally
-subjective: what one likes, another detests. However, problems solved by code
-are logical. We may not agree on "good", but **we can objectively and
-dispassionately characterize code**. For example:
+Software is controversial. "Good" is subjective: what one likes, another hates.
+However, code solves problems with logic. Consequently, we can **objectively
+and dispassionately characterize logic in terms of complexity**. For example:
```

Comment:
Then in the rest of it discuss the notion of _bottom_ (which is the notion of undefinedness or nothing which inevitably produces defects): http://typed.funops.co/tutorials/2016/05/25/unreasonable-effectiveness-of-reasoning-understanding-bottom.html